resilience4j/assets/configuration/spec.yaml

```
@@ -0,0 +1,10 @@
+name: Resilience4j
+files:
+- name: resilience4j.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/default
+  - template: instances
+    options:
+    - template: instances/default
```

```suggestion
    - template: instances/default
    - template: instances/openmetrics
      overrides:
        openmetrics_endpoint.value.example: http://localhost:9080/actuator/prometheus
        openmetrics_endpoint.description: |
          Endpoint exposing the Resilience4J prometheus metrics.
```
We can leverage the openmetrics config template to automatically inherit all the properties related to openmetrics checks.

Adding this will require running:
`ddev validate config resilience4j -s` and `ddev validate models resilience4j -s` to update the generated files and the `conf.yaml.example` files.